DOCUMENTATION.md

```diff
@@ -2,13 +2,13 @@
 
 - [Disclaimer](#disclaimer)
 - [Cards](#cards)
-    - [Effects](#effects)
-    - [Status](#status)
+	- [Effects](#effects)
+	- [Status](#status)
 - [Entity](#entity)
 - [Map](#map)
-    - [Rooms](#rooms)
-    - [Events](#events)
-    - [Light mechanic](#light-mechanic)
+	- [Rooms](#rooms)
+	- [Events](#events)
+	- [Light mechanic](#light-mechanic)
```

Comment:
any reason to add one more tab in the documentation ?

Reply:
No, they just kinda happen for some reason